analysis/data_utils.py

```diff
@@ -42,6 +39,43 @@ def drop_uninteresting_columns(experiment_df):
     ]]
 
 
+FuzzerBenchmark = collections.namedtuple('FuzzerBenchmark',
+                                         ['fuzzer', 'benchmark', 'experiment'])
+
+
+def clobber_snapshots(df, experiments):
```

Comment:
We should call this something else than "snapshots", because in the rest of the code by "snapshot" we refer to a "data frame where all time values are the same": https://github.com/google/fuzzbench/blob/master/analysis/README.md#snapshot.

How about calling it `clobber_experiments_data`, or something like that?

Reply:
I changed it to clobber_experiments_data to be consistent with the graphing code.
But as I understand it, the doc's explanation of the term 'snapshot' is inconsistent with the rest of the code base where snapshot refers to a particular time and a particular trial not a particular time across all trials. See the snapshot model for example: https://github.com/google/fuzzbench/blob/master/database/models.py#L49

Reply:
Yes, https://github.com/google/fuzzbench/blob/master/analysis/README.md is about the three types of DataFrames we use under analysis/. But yea, naming is hard :) we can think about better names